certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1614,6 +1624,97 @@ public class Script : ScriptBase
       response.Content = new StringContent(body.ToString(), Encoding.UTF8, "application/json");
     }
 
+    if ("GetRecipientFields".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase))
+    {
+      var body = ParseContentAsJObject(await response.Content.ReadAsStringAsync().ConfigureAwait(false), false);
+      List<JToken> recipientList = new List<JToken>();
+      recipientList.Add(body["signers"]);
```

Comment:
I would assign the signer types in an array. Loop through it and inside the loop add one line.
`recipientList.Add(body[signerType])`

Reply:
Done